src/Experimental/Extensions/Extensions.jl

```diff
@@ -12,14 +12,16 @@ Extensions
 
 import ..Formats
 import ..Cache
+import ..Utilities: AbstractConfig, EmptyConfig
+
 
 # Just add an entry in the metadata and return the stripped string.
-function Formats.metamacro{name}(::Formats.MetaMacro{name}, body, mod, obj)
+function Formats.metamacro{name}(::Formats.MetaMacro{name}, body, mod, obj; conf::AbstractConfig=EmptyConfig())
```

Comment:
I wanted to avoid adding here any kwargs but I got errors like:

```
ERROR: LoadError: Undefined metamacro.
 in extractmeta! at /home/workerm/.julia/v0.4/Docile/src/Formats/metaparse.jl:22
 ```

which i thought should not happen with the optional kwarg.

Any ideas?